cmake/projects/flex/hunter.cmake

```diff
@@ -18,6 +19,6 @@ hunter_add_version(
 )
 
 hunter_configuration_types(flex CONFIGURATION_TYPES Release)
-hunter_pick_scheme(DEFAULT url_sha1_autotools)
+hunter_pick_scheme(DEFAULT url_sha1_flex_autotools)
```

Comment:
Since sources haven't changed but they way package build changed you have to update `PACKAGE_INTERNAL_DEPS_ID`:
* https://docs.hunter.sh/en/latest/reference/user-modules/hunter_download.html

Reply:
I thought that might be the case.